packages/xml-views-validation/src/validators/elements/non-stable-id.ts

```diff
@@ -0,0 +1,123 @@
+import { some, includes } from "lodash";
+import { XMLElement } from "@xml-tools/ast";
+import {
+  getUI5ClassByXMLElement,
+  ui5NodeToFQN,
+  resolveXMLNS,
+} from "@ui5-language-assistant/logic-utils";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import { NonStableIDIssue } from "../../../api";
+import { NON_STABLE_ID, getMessage } from "../../utils/messages";
+import { isCustomClass } from "../../utils/custom-class";
+import { CORE_NS } from "../../utils/special-namespaces";
+
+export function validateNonStableId(
+  xmlElement: XMLElement,
+  model: UI5SemanticModel
+): NonStableIDIssue[] {
+  // Can't give an error if there is no position or name
+  if (xmlElement.name === null || xmlElement.syntax.openName === undefined) {
+    return [];
+  }
+
+  if (isWhiteListed(xmlElement, model)) {
```

Comment:
If all scenarios of the `whiteList` are UI5 classes edge cases
perhaps we can give it a more accurate name, e.g: `isWhiteListClass`
or maybe the more general `isWhiteListedTag` 
or `isTagExcludedFromStableIDCheck`?

WDYT?